ledger/accountdb.go

```diff
@@ -52,6 +52,12 @@ var accountsSchema = []string{
 		data blob)`,
 }
 
+var accountsResetExprs = []string{
```

Comment:
You might want to change this into `DROP TABLE IF EXISTS [tablename]` so that it won't generate an error if one of the tables are missing while the other exists.

Reply:
Done